files/common/usr/bin/get-property-from-image

```diff
@@ -53,49 +60,49 @@ shift $((OPTIND - 1))
 [[ $# -gt 2 ]] && usage "too many arguments specified"
 [[ $# -lt 2 ]] && usage "too few arguments specified"
 
-[[ "$EUID" -ne 0 ]] && die "must be run as root"
-[[ -d "$UPDATE_DIR" ]] || die "$UPDATE_DIR does not exist"
+[[ "$EUID" -ne 0 ]] && die 1 "must be run as root"
```

Comment:
for all cases where `die 1 ...` is used, should we just use `die ...` since `1` is the default when not specified?

Reply:
We can, I don't have a preference. Will update the scripts.